internal/server/data/organization.go

```diff
@@ -1,50 +1,22 @@
 package data
 
 import (
-	"context"
 	"fmt"
 
-	"gorm.io/gorm"
-
 	"github.com/infrahq/infra/internal/server/models"
 	"github.com/infrahq/infra/uid"
 )
 
-type orgCtxKey struct{}
-
-// OrgFromContext returns the Organization stored using WithOrg.
-func OrgFromContext(ctx context.Context) *models.Organization {
-	org, ok := ctx.Value(orgCtxKey{}).(*models.Organization)
-	if !ok {
-		return nil
-	}
-	return org
-}
-
-func MustGetOrgFromContext(ctx context.Context) *models.Organization {
```

Comment:
Now that we have a strongly typed interface we don't need to store the org in the context. This wasn't necessary to do all at once, but it was a very small change at the end (see the last commit).

We no longer panic, but an unset orgID will also be 0, which will never match a real org (not even the default), which will still error and should be caught by a test.